lessons/bn/advanced/protocols.md

```diff
@@ -0,0 +1,113 @@
+%{
+  version: "1.0.1",
+  title: "প্রোটোকল",
+  excerpt: """
+  এই অধ্যায়ে, আমরা প্রোটোকল কি এবং কিভাবে এলিক্সিরে প্রোটোকল ব্যবহার করা হয় তা দেখবো।
+  """
+}
+---
+
+## প্রোটোকল 
+তো কি এই প্রোটোকল? 
+এলিক্সিরে পলিমরফিজম করার উপায় হলো প্রোটোকল।
+নতুন তৈরি টাইপস এর জন্যে কোনো এপিআই এর বর্ধন এরল্যাং এ প্রচুর কষ্টসাধ্য।
+তাই, এটা থেকে মুক্তি পেতে এলিক্সিরে ফাংশনকে তার ভ্যালু এর টাইপের ভিত্তিতে ডাইনামিকালি ডিসপ্যাচ করা হয়।
```

Comment:
Yes, it does! I'm a bit confused with the word "পাশ" and "পাস" though.  I think I read this word before in a book, It was something along the line "পাশ কাটিয়ে যাওয়া" which meant "এড়িয়ে যাওয়া" in that context. 
Let me know your thoughts.

Reply:
@wasi0013  bhai, sorry for begin late again. 😢 

Do you think "ফাংশনে প্রবেশ করানো ভ্যালু" sounds better? I suggested "পাশ" because, we often use the term, "passing an argument" so that felt more natural. And, I'm confused between "পাশ" and "পাস" as well. "পাশ" is a bengali word, that means "Side", not sure about "পাস" :/

Reply:
@m4hi2 bhai, I talked with one of my friends regarding this - "পাশ" and "পাস". They both have different origins and meanings. "পাশ"  is from "সংস্কৃত". The meaning of "পাশ" varies based on context and use case. On the other hand, "পাস" is originated from the English word "pass" is more appropriate for the current context. We can use it here.

Reply:
@wasi0013  bhai, then we should use "পাস" in this case. :D